broker-plugins/access-control/src/main/java/org/apache/qpid/server/security/access/config/AclRulePredicatesBuilder.java

```diff
@@ -123,6 +123,16 @@ else if (property == Property.ATTRIBUTES)
         {
             _attributeNames.addAll(splitToSet(value));
         }
+        else if (property == Property.CONNECTION_LIMIT)
+        {
+            LOGGER.warn("The ACL Rule property 'connection_limit' has been deprecated");
+            return false;
+        }
+        else if (property == Property.CONNECTION_FREQUENCY_LIMIT)
+        {
+            LOGGER.warn("The ACL Rule property 'connection_frequency_limit' has been deprecated");
```

Comment:
The log message is misleading. The feature was completely removed. Only syntax support was left behind

Reply:
I changed the message to be precise.